### PR TITLE
Int 1033 testing

### PR DESCRIPTION
**Pull request by petern-annotell**
Adds integration testing for creation of LidarsAndCameras sequences with IMU data and frame shutter times.

### CODE REVIEW

**Comment by petern-annotell** [comment]
WIP still to actually test that the compressed IMU and time estimator files are available to retrieve after creating the input.

**Review by alydh-annotell — APPROVED** [review]

**Comment by alydh-annotell** [comment]
Did you manage to fetch the compressed IMU data as well?

File: annotell-input-api/examples/lidars_and_cameras_seq_with_imu_and_shutter_times.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    cam_sensor2 = "RFC02"
+    metadata = {"location-lat": 27.986065, "location-long": 86.922623, "vehicleId": "abg"}
+    examples_path = os.path.dirname(__file__)
+
+    # Create calibration
+    calibration_spec = create_sensor_calibration(f"Collection {datetime.now()}", [lidar_sensor1, lidar_sensor2], [cam_sensor1, cam_sensor2])
+    created_calibration = client.calibration.create_calibration(calibration_spec)
+
+    # Generate IMU data
+    MILLI = 1000000 # nanos
```
Comment (Ragge-dev) [inline annotell-input-api/examples/lidars_and_cameras_seq_with_imu_and_shutter_times.py:37]: Can we rename this one to ONE_MILLISECOND or something similar? Got a bit tripped up before reading how it was used

File: annotell-input-api/examples/lidars_and_cameras_seq_with_imu_and_shutter_times.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                    ResourceModel.Image(examples_path + "/resources/img_RFC02.jpg", sensor_name=cam_sensor2,
+                                        metadata=ResourceModel.ImageMetadata(
+                                            shutter_time_start_ns=start_ts + 0.5 * MILLI,
+                                            shutter_time_end_ns=start_ts + 1.5 * MILLI))
+                ],
+            ),
+            LCSM.Frame(
+                frame_id="2",
+                unix_timestamp=start_ts + 5 * MILLI,
+                relative_timestamp=4 * MILLI,
```
Comment (Ragge-dev) [inline annotell-input-api/examples/lidars_and_cameras_seq_with_imu_and_shutter_times.py:67]: relative_timestamp is expressed in milliseconds according to docs. (I think it actual, but might be good to be consistent with out docs: https://annotell.github.io/annotell-python/docs/input-api/overview#frames-sequential-inputs

**Review by Ragge-dev — APPROVED** [review]
Have a look at comments, then you are free to merge:)

**Pull request merged**